common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -38,11 +38,8 @@ public final void init() {
 
         this.conditions = conditions.build();
 
-        if (this.conditions.isEmpty()) {
-            enable();
-        } else {
-            this.conditions.forEach(Condition::init);
-        }
+        if (this.conditions.isEmpty()) return;
```

Comment:
If there are no conditions, return? 
This used to be like: 

- no condition -> enable
- condition -> init conditions.

Reply:
but the old code looks odd. if you had conditions, they got inited, but the feature did not get enabled. why the asymmetry?

Reply:
it did get enabled, actually - it was just buried within the condition code itself

the point of changing this here was to separate the feature initialization (condition stuff, mainly) and feature enabling - in case we don't actually want to enable the feature immediately